index.html

```diff
@@ -14,6 +15,61 @@
 </head>
 
 <body>
-  <h1>My Resume</h1>
+</style>
+</head>
+
+
+
+<div id="header"></div>
+<div class="left"></div>
+<div class="stuff">
+  <br><br>
+  <img class="my-portrait" src="https://upload.wikimedia.org/wikipedia/en/e/e0/Gollum.PNG" style="width: 25%">
+  <h1>Sm&eacute;agol - Web Dev</h1>
+  <h2></h2>
+  <hr />
+  <br>
+
+  <p class="head">Introduction</p>
+  I need my Precious!
+  <br>
+  <br>
+
+
+  <p class="head">Skills</p>
+  <ul>
+    <li>Stoor Hobbit</li>
+    <li>Fisherman</li>
+    <li>Thief</li>
+    <li>Falling in Volcanos</li>
+  </ul>
+  <br><br>
+
+  <p class="head">Work History</p>
+  <ul>
+    <li>Killing Hobbitses</li>
+    <li>Hiding Precious</li>
+    <li>Running Away</li>
+    <li>Dying in a Volcano</li>
+  </ul>
+
+<br>
+  <section class="nav footer">
+
+    <a href="#">Home</a>
+    <a href="#">Intro</a>
+    <a href="#">Interests</a>
+    <a href="#">Contact</a>
+
+  </section>
+
+
+</div>
```

Comment:
You've got a nice simple design here, but it could use a bit more styling! I'd recommend looking into [HTML Tables](https://html.com/tables/) and the `:nth-child(even)` pseudo-selector for styling if you're curious about how it's handled in the design file.